doc/nrf/protocols/thread/device_types.rst

```diff
@@ -25,48 +29,50 @@ For additional information about Thread device types, see the documentation for
 Full Thread Devices
 *******************
 
-In order to maintain the routing table, FTDs keep their radio on at all times.
-An important difference between FTDs and MTDs is that FTDs subscribe to special "all-routers" multicast addresses (``ff02::2`` and ``ff03::2``, see `IPv6 multicast addressing`_).
-Because of these reasons, they typically consume more power than MTDs.
+FTDs keep their radio enabled at all times to be ready to receive messages asynchronously.
+They typically consume more power than MTDs and are mains-powered, which allows them to have the following functionalities that MTDs do not have, among others:
+
+* Ability to resolve IPv6 addresses
+* Handling address registration
+* Maintaining routing table
+* Maintaining links with neighbors
 
-Full Thread Devices can be further divided into three categories:
+Another important difference between FTDs and MTDs is that FTDs subscribe to special "all-routers" multicast addresses (``ff02::2`` and ``ff03::2``, see `IPv6 multicast addressing`_).
+
+Full Thread Devices can be further divided into the following categories, which depend on the topology of the network:
 
 * Router
-* Full End Device (FED)
-* Router Eligible End Device (REED)
+* Full End Device (FED) or Router Eligible End Device (REED)
 
-.. list-table:: Full Thread Device categories
+.. list-table:: Full Thread Device network role categories
    :widths: 15 10 10 15
    :header-rows: 1
 
    * -
      - Router
-     - Full End Device
-     - Router Eligible End Device
+     - Full End Device (FED) or Router Eligible End Device (REED)
    * - Maintains a routing table
      - Yes
      - Yes
-     - Yes
    * - Routes messages
      - Yes
      - No
-     - When working as a Router
    * - Can extend the network
      - Yes
      - No
-     - When working as a Router
 
 Routers can be regarded as the backbone of a Thread network.
 They maintain the routing table and forward messages to other devices.
 Routers, unlike End Devices, can also be used to extend the network range and a Router is required for a Minimal End Device to join the network.
-The maximum number of Routers in a single Thread network is 32.
+The maximum number of active Routers in a single Thread network is 32.
 
-FEDs maintain a routing table, but they can't route messages.
-This means they don't require a parent (Router) to function, but they cannot become parents themselves.
+FEDs are End Devices that maintain a routing table, but they can't route messages.
```

Comment:
```suggestion
FEDs are End Devices with FTDs capabilities to perform certain Thread management functionalities such as address resolution. Effectively reducing it's router computation power"
```

In last sentence.. i want to say that FED relieves its partent from doing things on behalf of the child. This is the main advantage of FED.